File: 프로그래머스/unrated/181856. 배열 비교하기/배열 비교하기.cs
using System;
using System.Linq;

public class Solution {
    public int solution(int[] arr1, int[] arr2) 
    {
        int a = 0;
        int b = 0;
        
        if(arr1.Length != arr2.Length)
        {
            a = arr1.Length;
            b = arr2.Length;
        }
        else
        {
            a = arr1.Sum();
            b = arr2.Sum();
        }
        
        if(a < b) return -1;
        if(a > b) return 1;
        return 0;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.cpp
#include <string>
#include <vector>
#include <numeric>

using namespace std;

int solution(vector<int> arr1, vector<int> arr2) {
    
    if (arr1.size() > arr2.size()) return 1;
    if (arr1.size() < arr2.size()) return -1;
    
    int sum1 = accumulate(arr1.begin(), arr1.end(), 0);
    int sum2 = accumulate(arr2.begin(), arr2.end(), 0);
    
    if (sum1 == sum2) return 0;
    return sum1 > sum2 ? 1 : -1;
}